mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/SinkOperations.java

```diff
@@ -0,0 +1,80 @@
+package org.apache.mesos.rx.java;
+
+import org.apache.mesos.v1.scheduler.Protos;
+import org.jetbrains.annotations.NotNull;
+import rx.functions.Action0;
+import rx.functions.Action1;
+
+public final class SinkOperations {
+    private SinkOperations() {}
+
+    @NotNull
+    public static SinkOperation create(
+        @NotNull final Protos.Call call,
+        @NotNull final Action0 onCompleted,
+        @NotNull final Action1<Throwable> onError
+    ) {
+        return new SinkOperation(call, onCompleted, onError);
+    }
+
+    @NotNull
+    public static SinkOperation create(
+        @NotNull final Protos.Call call,
+        @NotNull final Action0 onCompleted
+    ) {
+        return new SinkOperation(call, onCompleted, (t) -> {});
+    }
+
+    @NotNull
+    public static SinkOperation create(
+        @NotNull final Protos.Call call,
+        @NotNull final Action1<Throwable> onError
+    ) {
+        return new SinkOperation(call, () -> {}, onError);
+    }
+
+    @NotNull
+    public static SinkOperation create(
+        @NotNull final Protos.Call call
+    ) {
+        return new SinkOperation(
+            call,
+            () -> {},
+            (t) -> {}
+        );
+    }
+
+    @NotNull
+    public static SinkOperation sink(
```

Comment:
yeah, convenience with static imports.

It's convenient to be able to do `sink(..)` instead of `SinkOperations.sink(..)`.

Not super tied to it we can go with only one name if you think it'd be more clear.